.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
This shouldn't be part of the PR. 

Add `.vscode` in the `.gitignore` file and create another PR to solve this

Reply:
Sure